bouncer.go

```diff
@@ -273,52 +275,61 @@ func handleNoStreamCache(bouncer *Bouncer, rw http.ResponseWriter, req *http.Req
 		return
 	}
 	if bouncer.crowdsecMode == liveMode {
-		setDecision(bouncer.cache, remoteHost, true, int64(duration.Seconds()))
+		setDecision(remoteHost, true, int64(duration.Seconds()))
 	}
 }
 
 func handleStreamCache(bouncer *Bouncer) {
 	// TODO clean properly on exit.
+	// Instead of blocking the goroutine interval for all the secondary node,
+	// if the master service is shut down, other goroutine can take the lead
+	// because updated routine information is in the cache
+	_, err := getDecision(cacheTimeoutKey)
```

Comment:
What happens here, you try to check if the value exist in the cache ?

how about races conditions if 2 services check and write